### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by msdeshmukh009**
Landing page created for Orion store e-commerce app.

### CODE REVIEW

**Review by Varu98 — APPROVED** [review]
Nice Work on the comments to structure the code! 👍🏻

File: style.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  background-color: var(--sidebar-fallback-bg-color);
+}
+@supports (backdrop-filter: blur(3px)) {
+  .banner-content {
+    background-color: var(--banner-bg-color);
+    backdrop-filter: blur(3px);
+  }
+}
+.banner-heading {
+  font-family: "ultraregular", sans-serif;
```
Comment (mimansha-swarup) [inline style.css]: suggestion: u can make variable for this too

**Review by mimansha-swarup — APPROVED** [review]
🔥🔥🔥

**Review by manojasarsa — APPROVED** [review]
Looks nice. Good work!!

File: style.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  font-size: 2.5rem;
+}
+.title-hr {
+  border: 2px solid var(--primary-color);
+  background-color: var(--primary-color);
+  width: 70px;
+  height: 0;
+}
+.category-grid {
+  padding: 3rem 2rem;
```
Comment (Kavita539) [inline style.css:92]: suggestion: You can make global variables for spacings like padding, magin, etc. for easy accessibility and consistency in the code.

Comment (Kavita539) [reply style.css:92]: other than that it looks good.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  font-style: normal;
+}
+/***nav-bar***/
+.nav {
+  box-shadow: var(--nav-bar-shadow);
+}
+.nav-heading {
+  text-decoration: none;
+  color: var(--primary-color);
+  font-family: "ultraregular", sans-serif;
```
Comment (BelideNikhil) [inline style.css]: save font family & font-weight  as a variable as you are using it at multiple places,

File: style.css [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  justify-items: center;
+  align-items: center;
+}
+.card {
+  width: fit-content;
+}
+.overlay-text {
+  font-size: 1.2rem;
+  padding: 0.5rem;
+  border-radius: 0.3rem;
```
Comment (BelideNikhil) [inline style.css]: don't use REM for  border radius if it  is less than 8px, as it will shift-shape based on pixel density.

**Review by BelideNikhil — APPROVED** [review]
Great work!
the partly-blurred hero section looks cool!

File: style.css (hunk 3 above)
Comment (msdeshmukh009) [reply style.css]: I think I have a variable for font-weight forgot to use that. I will create variable for the font family.

File: style.css (hunk 4 above)
Comment (msdeshmukh009) [reply style.css]: Ok, I will make changes. Thanks for the review.

File: style.css (hunk 2 above)
Comment (msdeshmukh009) [reply style.css:92]: Ok, I will do that from now on. Thanks for the review!!

File: style.css (hunk 1 above)
Comment (msdeshmukh009) [reply style.css]: Will do that. Thanks for looking into the code!!

**Pull request merged**